Makefile

```diff
@@ -63,10 +63,10 @@ clean:
 # command. 'test' runs short tests that should last no more than a few seconds,
 # 'test-long' runs more thorough tests which should not last more than a few
 # minutes.
-pkgs = ./api ./compatibility ./crypto ./encoding ./modules/consensus	\
-       ./modules/gateway ./modules/host ./modules/hostdb		\
-       ./modules/miner ./modules/renter ./modules/transactionpool	\
-       ./modules/wallet ./modules/blockexplorer ./persist ./siad	\
+pkgs = ./api ./compatibility ./crypto ./encoding ./modules/consensus \
+       ./modules/gateway ./modules/host ./modules/hostdb		     \
+       ./modules/miner ./modules/renter ./modules/transactionpool    \
+       ./modules/wallet ./modules/blockexplorer ./persist ./siad     \
```

Comment:
oh. are your tabs set to 8 chars?

I just lined everything up to be next to the longest this time, instead of always putting it at 81chars like before.

But the GH 8-char-tabs mean that one guy is bumped out more